examples_tests/41.VisibilityBuffer/main.cpp

```diff
@@ -1191,6 +1192,23 @@ int main()
         }
     };
 
+    using DPG = ext::DepthPyramidGenerator::DepthPyramidGenerator;
+
+    DPG dpg(driver, am, depthBufferView);
+
+    const uint32_t mipCnt = DPG::getMaxMipCntFromImage(depthBufferView);
+    core::vector<core::smart_refctd_ptr<IGPUImageView>> mips(mipCnt);
+    DPG::createMipMapImageViews(driver, depthBufferView, mips.data());
+
+    core::smart_refctd_ptr<IGPUDescriptorSetLayout> dpgDsLayout;
+    core::smart_refctd_ptr<IGPUDescriptorSet> dpgDs;
+    auto dsLayoutDsTmp = DPG::createDescriptorSet(driver, depthBufferView, mips.data());
+    dpgDsLayout = dsLayoutDsTmp.first;
+    dpgDs = dsLayoutDsTmp.second;
+
+    core::smart_refctd_ptr<IGPUComputePipeline> dpgPpln;
+    dpg.createPipeline(driver, depthBufferView, mips.data(), dpgDsLayout, dpgPpln);
```

Comment:
what do you need the views for when creating a pipeline?

Reply:
i don't need those, i overlooked it